sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -200,7 +222,13 @@ internal class ViewAnnotationManagerImpl(
         if (isVisibleNow) {
           hiddenViewMap[inflatedView] = inflatedView.translationZ
           inflatedView.translationZ = mapView.translationZ - 1f
+          // view will actually be shown when we will restore translationZ
+          isShownNow = false
```

Comment:
Initially it was
`isShownNow = isVisibleNow`, but `if (isVisibleNow)` we set `isShownNow` to false. Seems `isShownNow` is always false, no?